### PR TITLE
[Common] util function to handle tensor format

### PR DESCRIPTION
**Pull request by jaeyun-jung**
Add util function to handle tensor format.

Signed-off-by: Jaeyun <jy1210.jung@samsung.com>

### CODE REVIEW

**Review by myungjoo — APPROVED** [review]
LGTM

File: gst/nnstreamer/include/nnstreamer_plugin_api.h [hunk 1]
```diff
@@ -407,6 +407,20 @@ gst_tensor_get_type (const gchar * typestr);
 extern const gchar *
 gst_tensor_get_type_string (tensor_type type);
 
+/**
+ * @brief Get tensor format from string input.
+ * @param typestr The string format name, supposed to be one of tensor_format_name[].
```
Comment (gichan-jang) [inline gst/nnstreamer/include/nnstreamer_plugin_api.h:412]:
```suggestion
 * @param format_str The string format name, supposed to be one of tensor_format_name[].
```

Comment (jaeyun-jung) [reply gst/nnstreamer/include/nnstreamer_plugin_api.h]: updated

**Review by gichan-jang — APPROVED** [review]
LGTM

**Review by dongju-chae — APPROVED** [review]
LGTM

**Pull request merged**